.github/stale.yml

```diff
@@ -0,0 +1,15 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 365
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 90
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "Type: Feature"
```

Comment:
Let's add the "Type: Understood" tag to the list of exceptions.  This will provide us a way to keep specific issue which we know are real but haven't yet been addressed.